single.php

```diff
@@ -32,17 +32,19 @@
 									else:
 										echo_documents_cover();
 									endif;
+
+									if(get_the_content()):
+										echo "<p>".get_the_content()."</p>";
+									endif;
+
+									if(odm_echo_extras()):
+										odm_echo_extras();
+									endif;
+
+									if(echo_downloaded_documents()):
```

Comment:
@Huyeng the `echo_downloaded_documents` and `odm_echo_extras` functions do not return any value. Why are you then checking if it returns true? 

I believe you should just call it.